pkg/etcd/etcd.go

```diff
@@ -389,6 +389,8 @@ func (e *ETCD) cluster(ctx context.Context, forceNew bool, options executor.Init
 			ClientCertAuth: true,
 			TrustedCAFile:  e.config.Runtime.ETCDPeerCA,
 		},
+		ElectionTimeout:   5000,
```

Comment:
adjuste to customizable parameters?

Reply:
I'll address the customization of etcd in a follow up PR.  We will probably need to do an approach where you can specific the etcd conf file and we merge it, similar to containerd config.  There's to many params to add to the CLI to address everything.